finder-app/makefile

```diff
@@ -0,0 +1,13 @@
+# Author: Anshul Somani
+# This makefile builds an executable for writer.c for two different targets, native and aarch64-none-linux-gnu.
+
+writer: writer.c
+	gcc -Wall -Werror -o writer writer.c 
+
+clean:
+	rm -f writer
+	
+CROSS_COMPILE:
+	aarch64-none-linux-gnu-gcc -g -Wall -c -o writer.o writer.c
```

Comment:
@Anshul-99 what will happen here if you do `make all` ? or make with CROSS_COMPILE=aarch64-none-gnu-
CROSS_COMPILE is not target to compile as you've added in Makefile. target is writer.c which we have to compile for aarch64.